front/src/modules/project/components/AddOrEditProjectModal.tsx

```diff
@@ -216,12 +216,14 @@ export default function AddOrEditProjectModal({
       cleanLocalStorageByProject(project!.id, projectStudies, dispatch);
     }
 
+    // navigate before deleting because otherwise RTK fires a new request to fetch the deleted project
+    navigate(`/operational-studies/projects/`);
```

Comment:
Not sure how to do better, but here we navigate regardless of the success of the deletion. How would you do @emersion ?

Reply:
Unfortunately, it doesn't seem like there is a very good answer to this particular problem: https://github.com/reduxjs/redux-toolkit/issues/1672

Reply:
This topic is interesting. We already discussed some month ago about using the syntax mentionned in the comments.
https://redux-toolkit.js.org/rtk-query/usage/automated-refetching#invalidating-cache-data

This would be a larger refacto but as this issue isn't critical, this could be a good moment to take the time to test this logic in `osrdEditoastApi.ts` for these endpoints which aren't affecting a large part of the app.